src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -93,14 +93,9 @@ internal static Encoding GetDefaultEncoding()
         {
             if (s_defaultEncoding == null)
             {
-#if UNIX        // PowerShell Core on Unix
-                s_defaultEncoding = new UTF8Encoding(false);
-#else           // PowerShell Core on Windows
+                // load all available encodings
                 EncodingRegisterProvider();
-
-                uint currentAnsiCp = NativeMethods.GetACP();
-                s_defaultEncoding = Encoding.GetEncoding((int)currentAnsiCp);
-#endif
+                s_defaultEncoding = new UTF8Encoding(false);
```

Comment:
I don't like the change. Previously (discussion #3248, PR #3467) I fixed this for Windows because we're still very sensitive to the OEM encoding. We have tons applications in enterprises that doesn't work if we don't set the correct OEM encoding on the system. If MSFT wants to cut it off, do it first in Windows 10 kernel and get feedback.

Reply:
@iSazonov where are tests which provide validation for the changes you made in PR3467? The discussion in #3248 doesn't seem to have any concrete examples of misbehavior, which I would really like to understand. As for your last, I would rather get feedback on breaking behavior outside of the Windows ship cycles.

Reply:
@iSazonov, @PowerShell/powershell-committee has already decided to go with UTF8 (No BOM) as default.  Can you give me a specific example where this is known to break?  If OEM is needed, perhaps we could add a setting in the powershell json file but still default to utf8

Reply:
PR #3467#3467 haven't tests because (1) we did not have a full set of encoding tests in the time (2) we waited the encoding RFC (3) we only restore Windows PowerShell behavior by removing a breaking change.
About concrete examples. I speak about business applications. I have no idea how I can demonstrate it concrete. I understand that you hardly ever changed the OEM encoding on your computers. But ask yourself why are customers forced to do so? If MSTF wants to see it and understand it, it needs TAP with non-english enterprises. As I said earlier, the simple answer is that we have many business applications that only work with a specific OEM encoding and read/write files in OEM. PowerShell also works in _this environment_! The #3467 was caused by the fact that the scripts were no longer working and we needed to re-encode the files. The value of PowerShell Core disappears completely.

We can look at it on the other side. A year ago there was a categorical position that we should be backward compatible and that we should avoid making breaking changes as much as possible. The position was later changed so we want to facilitate the adaptation of UNIX users. This led to the creation of many incompatible changes. The current change from this series. These two principles have proved to be contradictory. We need to decide which one's in charge.

If the first means that we want to allow multiple Windows users to use PowerShell on Unix without much effort, and to get feedback to start the process of evolutionary convergence of platforms.

The second actually means that we're creating a brand new product that cannot be version 6, but must be PowerShell Core 1.0. It's fair to say that Windows users will have to migrate to a new product instead of using it directly. In this case, I agree that we can be ahead of the Windows 10 system and even its Notepad and use Utf8 by default like all other changes. Only then will it not be expected that the community will begin to use PowerShell Core extensively before version 3.
The second way is very dangerous. Microsoft can break the Windows PowerShell community and not build a new one-there are already well-known and popular alternatives (PHP, JS, Perl) that have proven their reliability and performance and have already formed a significant community. I've seen some examples of Microsoft already quickly destroying communities. The recovery took several years. It's sad. If it were my business, I would prefer to keep today's real customers, not future abstracts, and make the first step similar WSL like Windows PowerShell for Unix to capture a reliable bridgehead. Only then the community would be prepared to accept any breaking changes as understandable and necessary. Sorry for many words :-)

Reply:
It seems again that a viable option is to have a setting for powershell.runtimeconfig.json to have this be OEM for compat reasons.

Reply:
I haven't followed this closely, but what happened to the `$PSDefaultEncoding = 'WindowsLegacy'` proposal from the RFC?

Reply:
And, just to clarify: for backward compatibility, it's both OEM _and "ANSI"_ code pages that must be considered.

Reply:
@mklement0 the simplified version we scoped for 6.0.0 does not include `WindowsLegacy`

Reply:
@JamesWTruher can you open an issue to cover @iSazonov 's concern?

Reply:
@mklement0 WindowsLegacy was pulled waiting for customer need - opened https://github.com/PowerShell/PowerShell/issues/5204 to track.